js/gui/gui-layout.js

```diff
@@ -170,6 +170,65 @@ function createBGColorLayout(group) {
     group.appendChild(item);
 }
 
+//updates GUI everytime song is added / updated
+let songList = []
+
+function updateMusicList() {
+    const musicList = document.getElementById("musiclist")
```

Comment:
The code will fail currently because there is no `musicbox`, `musicboxbutton`, `musiclist` elements in our html page.
Also, I will suggest to create a separated file to handle music related logics.
I will suggest to separate this PR into 2 PRs, one PR for background Blur and one PR for music.